recipe/meta.yaml

```
@@ -1,16 +1,16 @@
 {% set name = "mdal" %}
-{% set version = "0.7.3" %}
+{% set version = "0.7.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/lutraconsulting/MDAL/archive/release-{{ version }}.tar.gz
-  sha256: da69b46dac5f42db451556fec7604ea494d41b6e3c83a775bffee1966506c8c5
+  url: https://github.com/lutraconsulting/MDAL/archive/0.7.4.tar.gz
```

To avoid getting the version number out of sync and also allow the bot to help with updates:
```suggestion
  url: https://github.com/lutraconsulting/MDAL/archive/{{ version }}.tar.gz
```

---

I don't agree

This was the original, but actually there have been inconsistencies in the URL between releases.

I also do not see the value - since the update for a new release has to be manually created and the sha256 has to be manually edited - I can so no problem and much more resilience in having the URL manually updated